nautobot/core/testing/views.py

```diff
@@ -529,13 +529,32 @@ def test_delete_object_without_permission(self):
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
         def test_delete_object_with_permission(self):
+            from nautobot.extras.models import Note, ObjectChange
+
             instance = self.get_deletable_object()
+            instance_note_pk_list = []
+            assigned_object_type = ContentType.objects.get_for_model(self.model)
+            if hasattr(self.model, "notes"):
+                notes = (
+                    Note.objects.create(
+                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 1"
+                    ),
+                    Note.objects.create(
+                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 2"
+                    ),
+                    Note.objects.create(
+                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 3"
+                    ),
+                )
+                for note in notes:
+                    note.validated_save()
```

Comment:
Slight overkill/performance hit to do both `.create()` and `.validated_save()`, especially since this is being done in every view test now.

```suggestion
                notes = (
                    Note(
                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 1"
                    ),
                    Note(
                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 2"
                    ),
                    Note(
                        assigned_object_type=assigned_object_type, assigned_object_id=instance.id, note="hello 3"
                    ),
                )
                for note in notes:
                    note.validated_save()
```